### PR TITLE
Update the GUI container tutorial to use new terminology and add update image

### PR DESCRIPTION
**Pull request by LennDG**
The [GUI tutorial](https://docs.godotengine.org/en/latest/tutorials/ui/gui_containers.html) uses the term "Size Flags" and an outdated image. This was a source of confusion for me when attempting to learn UI in Godot 4.
This PR updates that section to use "Sizing options" and adds the additional options that now exist for container sizing.

There are also SizeFlags used in the [class reference for containers](https://docs.godotengine.org/en/latest/classes/class_container.html) and in the code, these still have the old terminology so they don't need to be updated.

### CODE REVIEW

File: tutorials/ui/gui_containers.rst [hunk 1]
```diff
@@ -34,24 +34,30 @@ Example of *HBoxContainer* resizing children buttons.
 
 The real strength of containers is that they can be nested (as nodes), allowing the creation of very complex layouts that resize effortlessly.
 
-Size flags
+Sizing options
 ----------
```
Comment (mhilbrunner) [inline tutorials/ui/gui_containers.rst:38]: The underline needs to be updated to match the new title length.

**Review by mhilbrunner — COMMENTED** [review]
Looks good, left a comment. Also, please convert the image to WebP while you're at it :)

File: tutorials/ui/gui_containers.rst (hunk 1 above)
Comment (LennDG) [reply tutorials/ui/gui_containers.rst]: Done in commit [253f4ba](https://github.com/godotengine/godot-docs/pull/6636/commits/253f4ba9ed6daa057033191802890946e8cfb368)

**Review by skyace65 — APPROVED** [review]
Looks good, thanks! And congrats on your first merged PR!

**Comment by LennDG** [comment]
Thank you! Hopefully the first of many :)

**Pull request merged**